src/main/java/bio/terra/app/configuration/JdbcConfiguration.java

```diff
@@ -99,6 +99,8 @@ private void configureDataSource() {
         props.setProperty("password", getPassword());
         props.setProperty("maxTotal", String.valueOf(poolMaxTotal));
         props.setProperty("maxIdle", String.valueOf(poolMaxIdle));
+        props.setProperty("logExpiredConnections", "true");
```

Comment:
It this a temporary thing or do we want to add that to be configurable. E.g. should these be member variables in this class?

Reply:
I intend it to be temporary to debug the current problem. I'll put a comment in to make that clear.

Reply:
Ok, thought maybe but just wanted to check